src/components/TrafficList/TrafficListComponent.tsx

```diff
@@ -0,0 +1,353 @@
+import * as React from 'react';
+import { Badge, Title, Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { style } from 'typestyle';
+import {
+  IRow,
+  ISortBy,
+  sortable,
+  SortByDirection,
+  Table,
+  TableBody,
+  TableHeader,
+  cellWidth
+} from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { TrafficItem, TrafficNode, TrafficDirection } from './TrafficDetails';
+import * as FilterComponent from '../FilterList/FilterComponent';
+import { ThresholdStatus, NA } from 'types/Health';
+import { NodeType, hasProtocolTraffic, ProtocolTraffic } from 'types/Graph';
+import { getTrafficHealth } from 'types/ErrorRate';
+import history, { URLParam } from 'app/History';
+import { createIcon } from 'components/Health/Helper';
+import { sortFields } from './FiltersAndSorts';
+import { SortField } from 'types/SortFilters';
+
+export interface TrafficListItem {
+  direction: TrafficDirection;
+  healthStatus: ThresholdStatus;
+  icon: string;
+  node: TrafficNode;
+  protocol: string;
+  trafficRate: string;
+  trafficPercent: string; // percent success
+}
+
+type TrafficListComponentProps = FilterComponent.Props<TrafficListItem> & {
+  trafficItems: TrafficItem[];
+};
+
+type TrafficListComponentState = FilterComponent.State<TrafficListItem> & {
+  sortBy: ISortBy;
+};
+
+const columns = [
+  {
+    title: 'Status',
+    transforms: [sortable, cellWidth(10)]
+  },
+  {
+    title: 'Name',
+    transforms: [sortable, cellWidth(30)]
+  },
+  {
+    title: 'Rate',
+    transforms: [sortable, cellWidth(12)]
+  },
+  {
+    title: 'Percent',
```

Comment:
I see this column was shortened from `Percent success` to only `Percent`.
As it is, I find it a little unclear what's the meaning of "Percent".

Any objection on going back to `Percent success`?

Reply:
+1 on this, as we have two percents (success or traffic distribution) I guess we should differentiate.

For example, if I come from graph, in the review service I may expect to see the split % instead of the percentage error.
![image](https://user-images.githubusercontent.com/1662329/94538481-5c081f80-0244-11eb-89f0-266c95f81725.png)

Reply:
+1, will change back to Percent Success.